.github/workflows/ci-slow.yml

```
@@ -112,6 +112,15 @@ jobs:
         run: |
           uv pip install --system alembic
           bash scripts/check-alembic-branches.sh
+      - name: Check .gitignore
+        run: |
+          if [ -f ".gitignore" ]; then
+            if grep -q -E "(^|\/)dashboard($|\/)" ".gitignore" || grep -q -E "(^|\/)src\/zenml\/zen_server\/dashboard($|\/)" ".gitignore"; then
+              echo "Error: The '/dashboard' or 'src/zenml/zen_server/dashboard' directory is ignored by Git."
+              echo "Please remove the corresponding entries from the .gitignore file."
+              exit 1
+            fi
+          fi
```

Or just install the dashboard?

```suggestion
      - name: Install latest dashboard
        run: bash scripts/install-dashboard.sh
```

---

+1